oracle-plugin/widgets/Oracle-connector.json

```diff
@@ -64,6 +64,29 @@
           "widget-type": "password",
           "label": "Password",
           "name": "password"
+        },
+        {
+          "label": "Role",
+          "name": "role",
+          "widget-type": "radio-group",
+          "widget-attributes": {
+            "layout": "inline",
+            "default": "normal",
+            "options": [
```

Comment:
Are these the only available roles in oracle?

Reply:
Yes.